docs/guides/UpgardeGuide-v5/Changes.md

```diff
@@ -0,0 +1,325 @@
+  - `Terra.should.beAccessible()` -> `Terra.it.isAccessible()`
+  - `Terra.should.matchScreenshot()` -> `Terra.it.matchesScreenshot()`
+  - `Terra.should.validate()` -> `Terra.it.validatesElement()`
+
+`Terra.it.isAccessible()` test options include:
+  - rules - the axe rules to assert in the test run
+  - viewports - this is not a recommended option. This can cause unintended behavior in UI and result in x page resizes for each Mocha `it` block rather than once before the test spec is ran.
+
+This helper no longer accept context as a test option but instead always use `document`. Delete from passed test options. i.e. `Terra.should.beAccessible({ context })` -> `Terra.it.isAccessible()`.
+
+`Terra.it.matchesScreenshot()` test options include:
+  - mismatchTolerance - percentage of mismatch acceptable before failure
+  - selector - the element to take a screenshot of
+  - viewports - this is not a recommended option. This can cause unintended behavior in UI and result in x page resizes for each Mocha `it` block rather than once before the test spec is ran.
+
+`Terra.it.validatesElement()` test options include:
+  - axeRules - the axe rules to assert in the test run
+  - mismatchTolerance - percentage of mismatch acceptable before failure
+  - selector - the element to take a screenshot of
+
+This helper no longer uses the selector value for scoping accessibility, but instead always use `document`.
+
+Added `Terra.validates` test helpers. These are chia test helpers that can be used inside of `it` blocks. Terra.it test helpers use these directly, but providing these allow for one to write more cohesive test specs.
+  - Terra.validates.accessibility()
+  - Terra.validates.screenshot()
+  - Terra.validates.element()
+
+Added the `Terra.describeViewports` test helper. This is a Mocha `describe` block which is intended to use used as a top-level describe block for tests to simplify test writing and enabled writing test that support both viewport and formFactor testing for better local testing and paralyzed CI testing.
+
+Using this will eliminate the need to use `--formFactors` when using the `tt-wdio` runner to run locale & form factor test locally--which improves test times.
+
+
+### ServeStaticService
+
+The serve static service can serve a static site or compile a site from the webpack config. The compiled site will be served by webpack-dev-server and the static site will be served by terra's serve static server.
+
+The service will no longer inject the locale into served html files so when serving static sites, you will be responsible for adding the locale to the static files.
+
+When compiling a sites with webpack, the ```defaultLocale```` environment variable will be passed to the webpack config for the current test locale. This will be done automatically for projects using terra-dev-site.
```

Comment:
```suggestion
When compiling a site with webpack, the `defaultLocale` environment variable will be passed to the webpack config for the current test locale. This will be done automatically for projects using terra-dev-site.
```